server/config/buildTables.js

```diff
@@ -1,14 +1,26 @@
 import pool from '.';
 
+const { log } = console;
+
 const userTableQuery = `
   CREATE TABLE IF NOT EXISTS users(
     id SERIAL PRIMARY KEY,
-    first_name VARCHAR(100) NOT NULL,
-    last_name VARCHAR(100) NOT NULL,
-    user_name VARCHAR(100) NOT NULL,
+    username VARCHAR(100) UNIQUE NOT NULL,
+    firstname VARCHAR(100) NOT NULL,
+    lastname VARCHAR(100) NOT NULL,
     email VARCHAR(100) UNIQUE NOT NULL,
-    password TEXT NOT NULL
-  )
+    password TEXT NOT NULL,
+    role int DEFAULT 0,
+    created_on TIMESTAMPTZ DEFAULT now() NOT NULL
+  );
+  CREATE TABLE IF NOT EXISTS books(
+    id SERIAL PRIMARY KEY,
+    title VARCHAR(100) UNIQUE NOT NULL,
+    body VARCHAR(100) NOT NULL,
+    description VARCHAR(100) NOT NULL,
```

Comment:
Good job @obayomi96 , what's the reason behind making the description field to accept just 100 characters? won't it be better if it can accept more than that since it is a book's description? or better still you can make a TEXT field.

Reply:
Thanks for pointing that out. I will rectify it.

Reply:
cool